hack/move-upstream.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# See HACKING.md for usage
```

Comment:
Kind of asking rhetorically since it's clear that's what it does -- just saying, it needs a little doc.